### PR TITLE
Added Swedish holidays + tests

### PR DESCRIPTION
**Pull request by martinlindhe**
+ some minor cleanups

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,4 @@
 vendor
+composer.lock
```
Comment (elseym) [inline .gitignore]: why should composer.lock not be under version control?

Comment (martinlindhe) [inline .gitignore]: it is autogenerated from composer.json when you install dependencies, so i dont see why it should be in the repository. but if you prefer it that way i can re-add it

Comment (globin) [inline .gitignore]: As this is a library i agree that there should be no composer.lock in the repository. For an application you always want a composer.lock in the repository which specifies your exact dependency's versions.

File: .gitignore [hunk 2]
```diff
@@ -1 +1,4 @@
...
+.idea
```
Comment (globin) [inline .gitignore]: Please use a global gitignore locally. We don't care what IDE someone uses and you would want this ignored in all of your projects. (https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)

**Comment by globin** [comment]
Apart from the nitpicking with the gitignore this looks good to me. I'll test this tomorrow morning.

Tack så mycket :beers:

**Comment by martinlindhe** [comment]
Okay, updated

**Pull request merged**